prettyprinter/src/Prettyprinter/Internal.hs

```diff
@@ -347,6 +350,54 @@ instance Pretty Char where
     prettyList = vsep . map unsafeTextWithoutNewlines . T.splitOn "\n"
 #endif
 
+-- | This class is similar to 'Pretty', but allows you to embed annotations in
+-- the 'Doc'.
+--
+-- @since 1.7.1
+class PrettyAnn a ann where
```

Comment:
To me it seems that the parameter should be ordered the other way round:
````haskell
class PrettyAnn ann a
````
In typical uses, I imagine the user chooses a fixed `ann` and then implements `PrettyAnn` instances for various `a`s.
It feels more natural to have the fixed parameter first and then the variable one.

For example see all the instances you define below.  
You would get a natural alignment for all the
```haskell
instance PrettyAnn ann ...
```

Reply:
@andreasabel Thanks for the feedback! You're absolutely right; That is better!